app/testOnly/services/TestDataXMLGenerator.scala

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package testOnly.services
+
+import cats.data.NonEmptyList
+import models.Arrival
+import models.ArrivalId
+import models.ChannelType
+import models.MessageStatus
+import models.MovementMessageWithStatus
+import models.MovementReferenceNumber
+import models.ArrivalStatus.Initialized
+import models.MessageType.ArrivalNotification
+import testOnly.models.SeedEori
+import testOnly.models.SeedMrn
+
+import java.time.Clock
+import java.time.LocalDateTime
+import javax.inject.Inject
+import scala.xml.Elem
+import scala.xml.XML
+
+private[services] class TestDataGenerator @Inject()(clock: Clock) {
+
+  def arrivalMovement(eori: SeedEori, mrn: SeedMrn, arrivalId: ArrivalId): Arrival = {
+
+    val dateTime = LocalDateTime.now(clock)
+
+    val xml = TestDataXMLGenerator.arrivalNotification(mrn.format, eori.format)
+
+    val movementMessage = MovementMessageWithStatus(dateTime, ArrivalNotification, xml, MessageStatus.SubmissionPending, 1)
+
+    Arrival(arrivalId,
+            ChannelType.web,
+            MovementReferenceNumber(mrn.format),
+            eori.format,
+            Initialized,
```

Comment:
When the data is inserted will there be a way to transition out of Initialised? Do we plan on using the test-support service with these users?

Reply:
> When the data is inserted will there be a way to transition out of Initialised?

That's a great spot Eric! No there is nothing that will interact with these to transition the state from Initialized after this data is seeded. I'll update this to be ArrivalSubmitted

> Do we plan on using the test-support service with these users?

No plans currently but we can use it on an ad-hoc basis going forward, which means that the status needs to be ArrivalSubmitted as you pointed out above

I'll get this fixed and updated asap